src/main/kotlin/org/jboss/tools/intellij/kubernetes/utils/Notification.kt

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
```

Comment:
Is this class still used ?

Reply:
No, it's not used any more. I thought about keeping it for future use. But then, non-used code is dead code, should be removed. I'll remove it.